packages/cli/src/commands/deploy/baremetal/SshExecutor.js

```diff
@@ -12,13 +12,14 @@ export class SshExecutor {
   async exec(path, command, args) {
     let sshCommand = command
 
+    const argsString = args?.join(' ') || ''
     if (args) {
```

Comment:
```suggestion
    if (argsString) {
```

Doesn't make a difference, but to me it feels like the code is easier to follow if you first check the variable you're going to use, and then use it. Rather than checking var A and then use var B

Reply:
Or you could now just get rid of the `if` statement and just always use `argsString` I guess :) Might add an extra space there, but I don't think that matters

Reply:
So:

```
const argsString = args?.join(' ') || ''
const sshCommand = command + argsString
```